### PR TITLE
Adds PHP Client playground

### PR DESCRIPTION
**Pull request by TomKlotzPro**
| Q                 | A
| ----------------- | ----------
| Bug fix?          | no
| New feature?      | no
| BC breaks?        | no     
| Related Issue     | [#1](https://github.com/algolia/api-clients-playground/issues/1)
| Need Doc update   | no

## Describe your change

Adding the php playground for algolia search client. 

This is related to this issue : [#1](https://github.com/algolia/api-clients-playground/issues/1)

### CODE REVIEW

**Comment by TomKlotzPro** [comment]
need a review @nunomaduro

File: php/LICENSE.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) Nuno Maduro <enunomaduro@gmail.com>
```
Comment (nunomaduro) [inline php/LICENSE.md]: Algolia Team <support@algolia.com>

File: php/README.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+## Installation
+
+> **Requires [PHP 7.2+](https://php.net/releases/)**
+
+```bash
+git clone https://github.com/algolia/algoliasearch-client-php-playground
```
Comment (nunomaduro) [inline php/README.md]: This is wrong.

File: php/README.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+```
+
+```bash
+cd algoliasearch-client-php-playground && composer update
```
Comment (nunomaduro) [inline php/README.md]: This is wrong.

Comment (nunomaduro) [reply php/README.md]: `composer install`.

File: php/composer.json [hunk 4]
```diff
@@ -0,0 +1,23 @@
+{
+    "name": "algolia/algoliasearch-client-php-playground",
```
Comment (nunomaduro) [inline php/composer.json]: algolia/api-clients-playground

File: php/composer.json [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+    "keywords": ["algolia", "php", "playground"],
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Nuno Maduro",
```
Comment (nunomaduro) [inline php/composer.json]: Algolia Team

File: php/composer.json [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+            "email": "enunomaduro@gmail.com"
```
Comment (nunomaduro) [inline php/composer.json]: support@algolia.com

**Comment by TomKlotzPro** [comment]
Need a review @nunomaduro

**Pull request merged**